modules/ROOT/pages/values-and-types/lists.adoc

```diff
@@ -292,7 +292,7 @@ RETURN keanu.resume
 
 It is not, however, possible to store heterogeneous lists as properties.
 For example, the following query, which tries to set a list including both the `title` and the `released` properties as the `resume` property of `Keanu Reeves` will fail.
-This is because the `title` property values are stored as strings, while the `released` property values are stored as integers.
+This is because the `title` property values are stored as `STRING` values, while the `released` property values are stored as `INTEGER`.
```

Comment:
```suggestion
This is because the `title` property values are stored as `STRING` values, while the `released` property values are stored as `INTEGER` values.
```